site/docs/v1/tech/installation-guide/securing.adoc

```diff
@@ -37,3 +37,32 @@ FusionAuth App to make requests to the search index.
 To access the FusionAuth UI you'll need to open port `9011`. If you have more than one instance of FusionAuth installed you will need
 to ensure that each instance of FusionAuth Backend can communicate on port `9011`.
 
+== Custom Certificates
+
+FusionAuth ships with the default `cacerts` file from OpenJDK. There are times when you need to add a certificate to the keystore. This might occur if:
+
+* You are using a self signed certificate.
+* Your certificate authority does not ship with the version of Java that FusionAuth uses.
+
+[WARNING.warning]
+====
+This operation modifies the security of your FusionAuth installation. Ensure that you know the provenance of this certificate and trust the issuer.
+====
+
+In some cases, such as with link:/docs/v1/tech/events-webhooks/securing/[webhooks], you may provide your own certificate via the FusionAuth API or UI, but you may need to install the certificate globally. You'll have to import this certificate in every FusionAuth installation.
+
+To add this, we need to use `keytool` to augment the Java keystore. First, find the Java installation that is running FusionAuth. This is typically at `FA_INSTALL_DIR/java/` where `FA_INSTALL_DIR` is where you installed FusionAuth. This is often at `/usr/local/fusionauth` but may vary.
+
+For this documentation, if your Java installation is at `/usr/local/fusionauth/java/current/`, the `/usr/local/fusionauth/java/current/lib/security/cacerts` file should be modified. Let's also assume the new certificate authority public key file is located at `newcacert.pem`, and that you haven't changed your keystore password from the default value, which is `changeit`.
```

Comment:
> (...)which is `changeit` (please remember to change this).

Not sure if this is a security best practice?

Reply:
Yep, per. IBM :) 

> https://www.ibm.com/docs/en/sdk-java-technology/8?topic=certificate-cacerts-certificates-file
> A certificates file named cacerts resides in the security properties directory, java.home\lib\security, where java.home is the runtime environment directory (the jre directory in the SDK or the top-level directory of the Java™ 2 Runtime Environment).
> 
> The cacerts file represents a system-wide keystore with CA certificates. System administrators can configure and manage that file using keytool, specifying jks as the keystore type. The cacerts keystore file ships with several root CA certificates. The initial password of the cacerts keystore file is changeit. System administrators should change that password and the default access permission of that file when installing the SDK.
> Important: Verify your cacerts file. Since you trust the CAs in the cacerts file as entities for signing and issuing certificates to other entities, you must manage the cacerts file carefully. The cacerts file should contain only certificates of the CAs you trust. It is your responsibility to verify the trusted root CA certificates bundled in the cacerts file and make your own trust decisions. To remove an untrusted CA certificate from the cacerts file, use the delete option of the keytool command. You can find the cacerts file in the runtime environment installation directory. Contact your system administrator if you do not have permission to edit this file.

Reply:
` jks as the keystore type` 

we could add a bit more about this, but not sure if that is too in the weeds
https://stackoverflow.com/questions/11536848/keystore-type-which-one-to-use/11540061#:~:text=JKS%2C%20Java%20Key%20Store.&text=This%20keystore%20is%20Java%20specific,used%20in%20other%20programming%20languages.

Reply:
I suppose we could change it on our end, since we're controlling the java install. Not sure this is the best place for that documentation, though.